node/config/def.go

```diff
@@ -131,6 +131,12 @@ type SealingConfig struct {
 
 	WaitDealsDelay Duration
 
+	// CommittedCapacityDefaultLifetime is the default duration a Committed Capacity (CC)
+	// sector will live before it must be extended or converted into sector containing deals
+	// before it is terminated.
+	// Value must be between 180-540 days inclusive.
+	CommittedCapacityDefaultLifetime Duration
```

Comment:
Should I consider including some validation around this value's range on-load? (I assume the actor protects its inputs and produces a runtime error, but I imagine catching that on startup might be a better experience?) 

If so, where is the best place to put this check? I quickly walked config init back to the node builders which had a smattering of config validations, but didn't seem like the right place for it. Suggestions?

Reply:
As noted in the other comment, we should check this in the precommit policy.

(we also need to set a default value below in this file)